lib/union/index.js

```diff
@@ -1,9 +1,6 @@
 import _curry2 from '../_internal/_curry2'
+import _set from '../_internal/_set'
 
 export default _curry2(function union(arr1, arr2) {
-  return arr1.concat(arr2).reduce(function(acc, item) {
-    return acc.indexOf(item) === -1
-      ? acc.concat(item)
-      : acc
-  }, [])
+  return _set(arr1).concat(arr2).values()
```

Comment:
`.values()` is not es5

Reply:
values are custom, see `_internal/_set/index.js`

Reply:
Sorry, you're right